### ISSUE TITLE
t/15config.t fails with perl 5.37.4

### ISSUE THREAD

**Issue by jplesnik** [duncs/clusterssh#153]
t/15config.t fails with perl 5.37.4:

```
#   Failed test 'returned ok'
#   at t/15config.t line 546.
#          got: 'die'
#     expected: 'return'

#   Failed test 'Expecting no STDERR'
#   at t/15config.t line 550.
#          got: ''
#     expected: 'Unable to write default $HOME/.clusterssh/config: Is a directory
# 
# '
# Looks like you failed 2 tests of 155.
t/15config.t ............. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/155 subtests 
```

More  info on https://github.com/Perl/perl5/issues/20376

**Comment by tmancill** [comment]
I believe this is the same issue addressed in #150.

**Comment by jplesnik** [comment]
You're right. This PR solves this error.

**Comment by duncs** [comment]
PR merged

**Comment by eserte** [comment]
@duncs: Can we also get a CPAN release? In the meantime, perl 5.38 and 5.40 were released, and these perl versions do not work with the latest App-ClusterSSH release.

**Comment by duncs** [comment]
I have an interesting case where running a test locally works:

> $ COVERAGE=1 prove -v -I lib t/pod-coverage.t 
> t/pod-coverage.t .. 
> 1..12
> ok 1 - Pod coverage on App::ClusterSSH
> ok 2 - Pod coverage on App::ClusterSSH::Window
> ok 3 - Pod coverage on App::ClusterSSH::Base
> ok 4 - Pod coverage on App::ClusterSSH::Range
> ok 5 - Pod coverage on App::ClusterSSH::Cluster
> ok 6 - Pod coverage on App::ClusterSSH::Getopt
> ok 7 - Pod coverage on App::ClusterSSH::L10N
> ok 8 - Pod coverage on App::ClusterSSH::Host
> ok 9 - Pod coverage on App::ClusterSSH::Config
> ok 10 - Pod coverage on App::ClusterSSH::Helper
> ok 11 - Pod coverage on App::ClusterSSH::Window::Tk
> ok 12 - Pod coverage on App::ClusterSSH::L10N::en
> ok
> All tests successful.
> Files=1, Tests=12,  1 wallclock secs ( 0.03 usr  0.00 sys +  0.14 cusr  0.05 csys =  0.22 CPU)
> Result: PASS

However, the tests I am setting up with Github Actions (to replace TravisCI) are failing:
  https://github.com/duncs/clusterssh/actions/runs/11353410148/job/31578344731

When I have worked this out, I will look at doing a CPAN release

**Comment by duncs** [comment]
I have released 4.18
